hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -923,6 +942,9 @@
             `extclk_ctrl.sel` CSR writes. In addition `extclk_ctrl.hi_speed_sel` CSR causes the
             divided clocks to step down. Verify this via SVA bound to clkmgr, and clock cycle
             counters.
+
+            Disable external clock source and verify the AST reliably falls back to the internal
```

Comment:
@tjaychen can we actually disable external clock at runtime without a reset? I am not sure anymore, but at least for the LC actuated one, we can only enable it, but not disable.

Reply:
Yes, the clkmgr can enable and disable extclk unless extclk_regwen is clear, and notice extclk_regwen can only be set high by reset (because rw0c).